app/wp-content/themes/kstg/library/js/vehicle.js

```diff
@@ -3,8 +3,18 @@ var $galleryImages = $('.carousel-images ul li');
 var $prev = $('.carousel .prev');
 var $next = $('.carousel .next');
 var counter = 0;
+var $calloutValue = $('.specialFeature').text();
```

Comment:
Nitpick: I wouldn't include a '$' with this variable name because you're not actually storing the element, just the text. Could be confusing later because you would likely thing that `$calloutValue` is giving you a jQuery element.